sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -201,7 +201,11 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
     // scalastyle:on line.size.limit
 
     if (PROCESS_TABLES.testingVersions.isEmpty) {
-      logError("Fail to get the latest Spark versions to test.")
+      if (PROCESS_TABLES.isPythonVersionAtLeast37) {
+        logError("Fail to get the latest Spark versions to test.")
+      } else {
+        logError("Python version <  3.7.0, the running environment is unavailable.")
```

Comment:
It seems that this PR shows `Python version <  3.7.0` when there is no python executable. Did I understand correctly?

Reply:
Yes, you understand right, this is exactly what I want to discuss：

Should we only check the availability of python3, regardless of the version, or should we specify the minimum available version?

I set the condition that `HiveExternalCatalogVersionsSuite` is allowed to be tested as `isPythonVersionAtLeast37 ` in this pr due to found `[Spark runs on Java 8/11/17, Scala 2.12/2.13, Python 3.7+ and R 3.5+.](https://github.com/apache/spark/blob/master/docs/index.md)` in document , so I also made `Python version < 3.7.0` clear in this message.  Under this condition, if the user uses `Python 3.0 ~ Python 3.6` for testing, they will clearly know that the test is ignored because the python version is too low and not Python 3 is unavailable

Reply:
I'm not sure whether we need to make this restriction. If not, I can weaken this condition

Reply:
@dongjoon-hyun In addition, are you worried that `3.7.0` here is not easy to maintain?